src/components/ToastContainer.tsx

```diff
@@ -24,18 +24,20 @@ export const ToastContainer: React.FC<ToastContainerProps> = props => {
     >
       {getToastToRender((position, toastList) => {
         const swag = {
-          className: cx(
-            `${DEFAULT.CSS_NAMESPACE}__toast-container`,
-            `${DEFAULT.CSS_NAMESPACE}__toast-container--${position}`,
-            { [`${DEFAULT.CSS_NAMESPACE}__toast-container--rtl`]: rtl },
-            parseClassName(className)
-          ),
+          className:
+            typeof className === 'function'
+              ? className({})
```

Comment:
@rhdeck shouldn't we pass the default className here ?

Reply:
Done